DeployJenkinsfile

```diff
@@ -20,156 +21,252 @@ pipeline {
+                    {
+                        dir('uaa-k8s-deploy')
+                        {
+                            // Check out repo with Dockerfiles and build/publish script
+                            git changelog: false,
+                                    credentialsId: 'DEVTOOLS_GITHUB_CREDENTIALS',
+                                    poll: false,
+                                    url: 'https://github.devtools.predix.io/predix-security/uaa-k8s-deploy.git',
+                                    branch: 'master'
+                        }
+                    }
                 }
+                stage('Update GitOps Repository')
+                {
+                    when {
+                        not { changeRequest() }
+                    }
+                    steps
+                    {
+                        script
+                        {
+                            if (params.DEPLOYMENT_TYPE == "cf3-release-candidate")
+                            {
+                                gitOpsEnvName = "rc"
+                            }
+                            else if(params.DEPLOYMENT_TYPE == "cf3-staging")
+                            {
+                                gitOpsEnvName = "stage"
+                            }
+                            else if(params.DEPLOYMENT_TYPE == "cf3-integration")
+                            {
+                                gitOpsEnvName = "int"
+                            }
 
-                script {
-                    def util = load('uaa/JenkinsfileCommon.groovy')
-                    APP_VERSION = util.getAppVersion()
-                    echo "Publishing UAA ${APP_VERSION} Artifact to Artifactory"
-                    def uploadSpec = """{
-                        "files": [
+                            sh """
+                            cd uaa-k8s-deploy/
+                            sed -ri "s/tag:.*/tag: ${params.GRID_ARTIFACTORY_IMAGE_TAG}/g" values-${params.envName}.yaml
+                            git add values-${params.envName}.yaml
+                            git status
+                            """
+                            
+                        } 
+                        
+                        script
+                        {
+                            withCredentials([gitUsernamePassword(credentialsId: 'DEVTOOLS_GITHUB_CREDENTIALS',gitToolName: 'git-tool')])
                             {
-                                "pattern": "deploy-workspace/cloudfoundry-identity-uaa-${APP_VERSION}.war",
-                                "target": "MAAXA/org/cloudfoundry/identity/cloudfoundry-identity-uaa/${APP_VERSION}/"
+                                sh """
+                                cd uaa-k8s-deploy
+                                git commit -m " Image tag updated to ${eksBuildImageTag} for ums-evolve service in ${params.envName} environment to trigger GitOps reconciliation by Jenkins Deploy Job ${BUILD_NUMBER}"
```

Comment:
Typo . Change message to "Image tag updated to ${params.IMAGE_TAG} for ${params.DEPLOYMENT_TYPE} environment to trigger GitOps reconciliation by Jenkins Deploy Job ${BUILD_NUMBER}"